scripts/remind_draft_preregistrations.py

```diff
@@ -69,8 +72,9 @@ def find_neglected_prereg_within_reminder_limit():
 
 @celery_app.task(name='scripts.remind_draft_preregistrations')
 def run_main(dry_run=True):
-    init_app(routes=False)
-    if not dry_run:
-        add_file_logger(logger, __file__)
-    main(dry_run=dry_run)
+    if not waffle.switch_is_active(features.OSF_PREREGISTRATION):
```

Comment:
It might seem better to waffle this in the Celery config in defaults but that causes a circular import in waffle

Reply:
Discussed with David M. in person and we can completely remove this reminder email now (no sense reminding people to finish their prereg draft when there's two weeks remaining...). Go ahead and remove the script, task, and email template.